.ci/semaphore.tests.sh

```diff
@@ -1,9 +1,13 @@
 set -e -x
 
-pip install coala-bears
+cp requirements.txt requirements.orig
+printf '%s\n%s\n%s\n%s\n' \
+    "git+https://github.com/coala/coala" \
+    "git+https://github.com/coala/coala-bears" \
+    "$(cat test-requirements.txt)" \
+    "$(cat requirements.txt)" \
+    > requirements.txt
 pip install -r requirements.txt
-pip install -r test-requirements.txt
-sed -i.bak '/bears = GitCommitBear/d' .coafile
```

Comment:
Can't we just fix to the latest dev-version inside `test-requirements.txt`? Then we don't introduce silent breakages by updates in the coala and coala-bears breakages without an explicit upgrade here^^

Reply:
Sound like a plan :)
Should I ammend this commit?

Reply:
Yes, @sladyn98 amend this commit.

Reply:
I am not very familiar with how the test-requirements have been written. But correct me if I am wrong these two lines would be added to `test - requirements.txt`
`git+https://github.com/coala/coala`
`git+https://github.com/coala/coala-bears`
And the above sh file would now contain the following 
```pip install -r requirements.txt```
```pip install -r test-requirements.txt```
```sed -i.bak '/bears = GitCommitBear/d' .coafile```

Reply:
Yeah, that should work @sladyn98